sql/signal_handler.cc

```diff
@@ -168,6 +169,7 @@ extern "C" sig_handler handle_fatal_signal(int sig)
 
   set_server_version(server_version, sizeof(server_version));
   my_safe_printf_stderr("Server version: %s\n", server_version);
+  my_safe_printf_stderr("Server source revision: %s\n", SOURCE_REVISION);
```

Comment:
combine these two my_safe_printf_stderr into a single function call and keep the line length below 80.